components/ui/LinkItem.jsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const LinkItem = ({ children, to, blank, underlined, inline }) => {
+  return (
+    <Link
+      href={to || "#"}
+      target={blank && "_blank"}
+      className={`w-max flex items-baseline gap-1 group cursor-pointer hover:no-underline underline-offset-2 ${
+        underlined && "underline"
+      } ${inline && "inline-flex"}`}
```

Comment:
```suggestion
      } ${inline ? "inline-flex": "flex"}`}
```

Also remove `flex` on line 9